### PR TITLE
fix: image centered

### LINKED ISSUE #142: Fix error styles at home page
**Describe the bug**
Home page looks good on my device. But in another devices its deforming. I will describe two problems and posible solutions

1. The delivery boy image when the screen is small vertically it goes off the screen, posible solution is add max-height and min-height. I have tested and for me, looks better with: `max-height: 60vh` and `min-height: 40vh`.

2. The delivery boy is bad positioned at the page. You're using a mix of `position: absolute, margin-top: 20%, margin-bottom: -40%, transform: translateY(-80%)` to position the image. I think that if you're using **position: absolute**, the best is to use their properties (left, right, top, bottom) to position the image. Posible solution: delete `margin-top: 20%, margin-bottom: -40%, transform: translateY(-80%)` properties and change it for the `top` property. I have tested and for me, looks better with: ` top: 25%`

3. The button "Aceptar" is bad positioned at the page. Looks good on my device but in another devices with more little screen (vertically) it is hidden (it is goes off the screen). I think you can use again `position: absolute, top: 35%` to have a relation with the image. Other solution is use grid / flex to center at the end and center

**Checklist**
- [x] **Delivery boy image**: Add min and max height
- [x] **Delivery boy image**: Remove the properties `margin-top: 20%, margin-bottom: -40%, transform: translateY(-80%)` and replace with some position property (top or bottom. I prefer top because with bottom the image is down when the screen is extended)
- [x] **"Aceptar" button**: Remove `transform: translate(0%, 1100%);` and replace it for other way to positioning the button

**Screenshots**
imagen

1. Go to https://dev--azordev-dasher-user.netlify.app/

### LINKED ISSUE #186: Image not centered and growing too much exponentially
**Describe the bug**
The dasher image of a delivery boy is aligned to left and gets bigger affecting the layout

**To Reproduce**

1. Go to landing site
2. Set the window width as less than 750px

**Screenshots**
localhost_3000_check (2)

**Additional context**

### LINKED ISSUE #187: Logo image affecting layout
**Describe the bug**
The logo image affects the layout on certain widths

**To Reproduce**

1. Go to landing page
2. Set, for example, width as 1300px width and 460px height

**Expected behavior**
I believe a good behavior for this would be for it to have a m

**Screenshots**
localhost_3000_check (4)

### PR DESCRIPTION
**Pull request by ildebr**
### Description

Made the MAinImage component get centered and set it to a max-height so it won't alter the layout.

fixes #187 
fixes #186
fixes #142

#### Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)

### How to test it

If applicable, describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

1. Go to landing site
2. change the window height and width

### Checklist:

- [ ] Doesn't break the current code.
- [ ] Passes linters and test, also is building.
- [ ] Doesn't have spelling or grammatical problems.
- [ ] Doesn't have unnecessary comments or debugging code.
- [ ] The branch is updated with the last dev branch changes.

### CODE REVIEW

**Comment by Israel-Laguan** [comment]
I tested as described on the issue #187 
image

I can see the same issue appearing

Regarding #186 seems like is centered

File: src/pages/Home/Home.styled.jsx [hunk 1]
```diff
@@ -52,6 +52,11 @@ export const ManImage = styled.img`
   position: absolute;
   transform: translateY(-80%);
 
+  position: relative;
+  left: 50%;
+  transform: translateY(-80%) translateX(-50%);
+  max-height: 410px;
```
Comment (heliomar-pena) [inline src/pages/Home/Home.styled.jsx]: Delete the line 53, the line 57 is overriding this line

**Review by heliomar-pena — CHANGES_REQUESTED** [review]
Not critical comment

**Review by heliomar-pena — CHANGES_REQUESTED** [review]
Please add styles to fix responsive error
imagen

**Review by heliomar-pena — COMMENTED** [review]
On small screens the title is hidden
imagen

The problem of the dasher user guy is fixed. Good job!
imagen

File: src/pages/Home/Home.styled.jsx [hunk 2]
```diff
@@ -32,15 +32,22 @@ export const HalfSection = styled.div`
     width: 100%;
     min-height: 100vh;
   }
+
+  &.images-section{
```
Comment (Israel-Laguan) [inline src/pages/Home/Home.styled.jsx]: Please add space before parenthesis

File: src/pages/Home/Home.styled.jsx [hunk 3]
```diff
@@ -51,7 +58,7 @@ export const ManImage = styled.img`
   object-fit: contain;
   position: absolute;
   transform: translateY(-80%);
-
+  max-height: 350px;
   position: relative;
```
Comment (Israel-Laguan) [inline src/pages/Home/Home.styled.jsx]: Position is repeated, can you check which one is correct and erase the other one, please?

File: src/pages/Home/Home.styled.jsx [hunk 4]
```diff
@@ -75,19 +82,10 @@ export const ManImage = styled.img`
     align-self: center;
     margin-top: 30%;
     height: 68vh;
-  }
-
-  @media (min-width: 960px) {
-    margin-top: 12%;
-  }
+    margin: 0;
```
Comment (Israel-Laguan) [inline src/pages/Home/Home.styled.jsx]: this overrides `margin-top` on line 83, can you verify if this is correct? If one rule is not needed please erase it

**Comment by heliomar-pena** [comment]
I will take this pull request to finish it

**Comment by Israel-Laguan** [comment]
Tested and looks fine!

**Review by Israel-Laguan — APPROVED** [review]

**Review by heliomar-pena — APPROVED** [review]

**Review by e-azocar — APPROVED** [review]

**Pull request merged**